src/psyclone/tests/domain/lfric/algorithm/lfric_alg_test.py

```diff
@@ -245,7 +246,8 @@ def test_kernel_from_metadata(lfric_alg):
     with pytest.raises(ValueError) as err:
         lfric_alg.kernel_from_metadata("not fortran", "john")
     assert ("Failed to find kernel 'john' in supplied code: 'not fortran'. "
-            "Is it a valid LFRic kernel?" in str(err.value))
+            "Is it a valid LFRic kernel? Original error was 'Parse Error: "
+            "Kernel type john does not exist'." in str(err.value))
```

Comment:
Due to change in `backend/visitor.py`?

Reply:
Yes, I found it useful to output the original error as well as it gave more context.